lib/src/rules/cascade_invocations.dart

```diff
@@ -178,12 +178,12 @@ class _CascadableExpression {
           isCritical: true);
     }
     // setters
-    return new _CascadableExpression._internal(
-        _getPrefixElementFromExpression(node.leftHandSide),
-        [node.rightHandSide],
-        canJoin: true,
-        canReceive: true,
-        canBeCascaded: true);
+    final VariableElement variable =
+        _getPrefixElementFromExpression(node.leftHandSide);
+    final canReceive = node.operator.type != TokenType.QUESTION_QUESTION_EQ &&
```

Comment:
No because this was not a simple identifier, so being an assignment expression means it is a setter accessed either with this or identifier.